### PR TITLE
Update Changelog to `99ccb31`

### PR DESCRIPTION
**Pull request by Noordfrees**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -339,6 +374,20 @@
...
 - CMakeLists.txt: Use a list for many build flags (#5328)
+- Only use embedded minizip if not installed (#5360, #5392)
+- Put binaries in correct locations when installing (#5359)
+- Enable Include Order Codecheck on Header Files (#5373)
+- Move definition of `WL_INSTALL_BINDIR` (#5383)
+- Remove `SYSTEM` from `target_compile_options` (#5396)
+- CMakeLists.txt: add custom uninstall target (#5358)
+- Resolve relative parts before uninstalling empty dirs (#5417)
+- Reformat version string (remove markdown format) (#5385)
+- Xenial's CMake version does not know `VERSION_GREATER_EQUAL`, use N`OT VERSION_LESS` instead
```
Comment (tothxa) [inline ChangeLog]:
```suggestion
- Xenial's CMake version does not know `VERSION_GREATER_EQUAL`, use `NOT VERSION_LESS` instead
```

File: ChangeLog [hunk 2]
```diff
@@ -232,6 +260,9 @@
 - Fix unhealthy usage of `split` in language menu (#5241)
 - Skip installing locales when translations are disabled (#5302)
 - Tut01 String Fix (#5327)
+- Fix string bugs in lore texts reported on Transifex (#5399)
+- Fix Transifex String Issues (#5462)
```
Comment (tothxa) [inline ChangeLog]: I'm not sure if these belong in I18n, they're just typo and punctuation fixes.

Comment (Noordfrees) [reply ChangeLog]: That's where we always put these entries, but I grouped it with another one now

File: ChangeLog (hunk 1 above)
Comment (Noordfrees) [reply ChangeLog]: fixed

File: ChangeLog [hunk 3]
```diff
@@ -224,14 +252,16 @@
 - Improved exception reporting (#5003)
     - Fix exception reporting in the game logic thread.
     - Adds `--verbose-i18n` flag.
-- Fix Transifex String Issues (#5072)
+- Fix Transifex String Issues (#5072, #5462)
```
Comment (tothxa) [inline ChangeLog]:
```suggestion
- Fix Transifex String Issues (#5072, #5399, #5462)
```

File: ChangeLog [hunk 4]
```diff
@@ -224,14 +252,16 @@
...
 - Fix Lua `(n|p)gettext` ignoring textdomains (#5080)
 - Properly format `size_t` type printf placeholder (#5096)
 - Tx String and CLI Help Fixes (#5179)
 - More Stringfixes (#5190)
 - Fix unhealthy usage of `split` in language menu (#5241)
 - Skip installing locales when translations are disabled (#5302)
 - Tut01 String Fix (#5327)
+- Fix string bugs in lore texts reported on Transifex (#5399)
```
Comment (tothxa) [inline ChangeLog]:
```suggestion
```

**Review by tothxa — COMMENTED** [review]
This one too. :smiley:

**Review by tothxa — APPROVED** [review]
Thank you.

**Pull request merged**